scripts/templates/template.yaml

```diff
@@ -20,3 +36,35 @@ objects:
         api.openshift.com/managed: "true"
     resourceApplyMode: sync
     resources: []
+# the SelectorSyncIdentityProvider is not something you can oc apply in-cluster, so is not created in `deploy/`
+- apiVersion: hive.openshift.io/v1alpha1
+  kind: SelectorSyncIdentityProvider
+  metadata:
+    name: osd-sre-identityprovider
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: "true"
+    identityProviders:
+    - challenge: true
+      ldap:
+        attributes:
+          email:
+          - ${IDENTITY_ATTR_EMAIL}
+          id:
+          - ${IDENTITY_ATTR_ID}
+          name:
+          - ${IDENTITY_ATTR_NAME}
+          preferredUsername:
+          - ${IDENTITY_ATTR_PREFERRED_USERNAME}
+        bindDN: ${IDENTITY_BIND_DN}
+        bindPassword:
```

Comment:
@jewzaam  Is this secure?

Reply:
it's a reference to a secret

Reply:
so it doesn't expose any password, just the name of the secret.  dno't see that being a big deal.  if you can get to the secret you can get to the oauth cr